src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,44 @@
+import classnames from 'classnames';
+import React, {
+  FunctionComponent,
+  useEffect,
+} from 'react';
+import { Errors } from '../../types/Errors';
+
+interface ErrorProps {
+  errorMessage: string,
+  setErrorMessage: React.Dispatch<React.SetStateAction<Errors>>;
+}
+
+export const ErrorNotification: FunctionComponent<ErrorProps> = ({
+  errorMessage,
+  setErrorMessage,
+}) => {
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setErrorMessage(Errors.None);
+    }, 3000);
+
+    return () => clearTimeout(timer);
+  }, []);
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classnames(
+        'notification is-danger is-light has-text-weight-normal',
+        { 'is-hidden': !errorMessage.length },
+      )}
+    >
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Disabling eslint - is not the best decision. Try to solve the error without it. If you have any questions, ask them in the fe_chat, please